deploy/runtime/defaults/Generic Battery_Commercial.txt

```diff
@@ -1794,7 +1794,7 @@ itc_fed_percent
 1
 1
 1
-26
+0
```

Comment:
Remind me why this one's going to 0?

Reply:
Generic system's defaults assume a thermal system, which is not subject to the ITC. Some users use generic battery for a stand-alone battery, but stand alone batteries also aren't subject to it. I figure it's better to make someone who is importing an existing renewable generation profile (like I did on the webinar) increase it, than to have it be too high for the two most common cases.